test/URDFactoryTest.sol

```diff
@@ -27,10 +27,11 @@ contract UrdFactoryTest is Test {
         emit EventsLib.UrdCreated(
             urdAddress, randomCaller, randomOwner, randomTimelock, randomRoot, randomIpfsHash, randomSalt
         );
-        address realAddress = factory.createUrd(randomOwner, randomTimelock, randomRoot, randomIpfsHash, randomSalt);
+        IUniversalRewardsDistributor urd =
+            factory.createUrd(randomOwner, randomTimelock, randomRoot, randomIpfsHash, randomSalt);
 
-        assertEq(realAddress, urdAddress);
-        IUniversalRewardsDistributor distributor = IUniversalRewardsDistributor(realAddress);
+        assertEq(address(urd), urdAddress);
+        IUniversalRewardsDistributor distributor = IUniversalRewardsDistributor(address(urd));
```

Comment:
This cast is no longer necessary

Reply:
I was not aware of that lol

Reply:
False for L33, true for L34

Reply:
Yes, github automatically highlights the 3 previous lines...